src/test/rustdoc/visibility.rs

```diff
@@ -0,0 +1,13 @@
+// compile-flags: --document-private-items
+
+#![crate_name = "foo"]
+
+// @has 'foo/fn.foo.html' '//pre' 'fn foo'
+// !@has 'foo/fn.foo.html' '//pre' 'pub'
+fn foo() {}
+
+mod bar {
+    // @has 'foo/bar/fn.baz.html' '//pre' 'fn baz'
+    // !@has 'foo/bar/fn.baz.html' '//pre' 'pub'
+    fn baz() {}
+}
```

Comment:
I would make this part of `pub(restricted)` instead.

Reply:
I merged the `pub-restricted` and `visibility` tests into just one named `visibility`.